src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -907,7 +929,23 @@ private void Dispose()
                     {
                         code = EngineUtils.SignalMap.Instance[sigName];
                     }
-                    _callback.OnException(thread, sigName, results.Results.TryFindString("signal-meaning"), code);
+                    bool stoppedAtSIGSTOP = false;
+                    if (sigName == "SIGSTOP")
```

Comment:
SIGSTOP/SIGCONT are used for job control. Not sure that SIGCONT would even continue a process that is not SIGSTOPed. Also not sure that you can detach from a process stopped with a SIGINT and not have it continue automatically.